cvat/schema.yml

```diff
@@ -790,6 +792,70 @@ paths:
                 items:
                   type: string
           description: A manifest content
+  /api/cloudstorages/{id}/content-v2:
+    get:
+      operationId: cloudstorages_retrieve_content_v2
+      summary: Method returns the content of the cloud storage
+      parameters:
+      - in: header
+        name: X-Organization
+        schema:
+          type: string
+      - in: query
+        name: delimiter
+        schema:
+          type: string
+        description: The character you use to group keys
```

Comment:
I don't understand this. What are keys? The word "key" doesn't occur anywhere else in this description.

Reply:
`key` is the official term of the file on the cloud bucket. Most parts of CS providers use this term.